creme/creme_core/views/generic/order.py

```diff
@@ -115,14 +126,13 @@ def post(self, request, *args, **kwargs):
 
             moved_instance = instances.pop(target_index)
 
-            new_order = min(new_order, len(instances))  # Resists to a race deletion
-            instances.insert(new_order - 1, moved_instance)
+            target_index = min(target_order - 1, len(instances))  # Resists to a race deletion
+            instances.insert(target_index, moved_instance)        # index 0..N-1
 
-            for index, instance in enumerate(instances, start=1):
+            for order, instance in enumerate(instances, start=1):
                 old_order = getattr(instance, order_fname)
 
-                if old_order != index:
-                    setattr(instance, order_fname, index)
-                    instance.save(force_update=True, update_fields=(order_fname,))
+                if old_order != order:
+                    self.update_order(instance, order_fname, order)
```

Comment:
Is this fix worth to be backported? (so it would need unit som tests with a model which is not ProductLine/ServiceLine)

Reply:
It is used in the views 'creme_config__reorder_menu_level0' & 'creme_config__reorder_instance' which are tested

Reply:
And it worth to be backported because it is a bug, and an old one !